package.json

```diff
@@ -64,7 +63,8 @@
     "split2": "^4.1.0",
     "swagger-parser": "^10.0.3",
     "tap": "^16.1.0",
-    "typescript": "^4.6.4"
+    "typescript": "^4.6.4",
+    "@types/node": "^16.7.1"
```

Comment:
It should be on top, as the other dependencies that starts with @

Reply:
Fixed